packages/builder/src/internal/cli/arguments-parser.ts

```diff
@@ -0,0 +1,278 @@
+import {
```

Comment:
Why we are not using a package to handle CMD arguments parsing? 
Example: https://www.npmjs.com/package/cli-params

Reply:
This is copy paste, not development from scratch.